Add TetensFormula saturation vapor pressure (#384)
NumericalEarth/Breeze.jl: AGENTS.md

   - Use `$(TYPEDSIGNATURES)` for automatic typed signature documentation (preferred over `$(SIGNATURES)`)
   - Never write explicit function signatures in docstrings; always use `$(TYPEDSIGNATURES)`
   - Add examples in docstrings when helpful

>    - **Citations in docstrings**: Use inline citations with `[Key](@citet)` syntax (renders as "Author (Year)" with link).

Reviewer: ```suggestion
   - **Citations in docstrings**: Use inline citations with `[Author1 and Author2 (year)](@cite Key)` or `[Author1 et al. (Year)](@cite Key)` syntax.
```
